### DISCUSSION TITLE
Verified Ticks To Writter

### DISCUSSION THREAD

**Discussion by SoumyaEXE** [forem/forem#19727] [Feature Requests]
Hey Whats About **Giving A Verified Tick To Dev Trusted Member** Who are working hard to make community healthy always

**Comment by Sarah-okolo** [comment]
Even for moderators as well, so we can Identify which authors on the platform are also moderators. I believe they give badges for these though, but you can only find it in the badges section in the authors profile, but I feel a small icon beside the authors name to show this would be more pleasant, as it could be easily identified by all, even without heading into the authors profile page.
